### PR TITLE
[tests-only][full-ci] Api test for user trying to set own personal space quota

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR adds the api test for user trying to set their own personal space quota.
The deleted scenarios are covered on `setQuota.feature` file
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L94
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L134
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L50
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L14

## Related Issue
Related to issue: https://github.com/owncloud/ocis/issues/5565

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpaces/spaceManagement.feature [hunk 1]
```diff
@@ -89,118 +89,6 @@ Feature: Space management
     And the json responded should not contain a space with name "Project"
     And the json responded should not contain a space with name "Alice Hansen"
 
-
-  Scenario: The space admin user changes the quota of the project space
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpaces/spaceManagement.feature:93]: deleted?

Comment (PrajwolAmatya) [reply tests/acceptance/features/apiSpaces/spaceManagement.feature:93]: These scenario got duplicated as it exits on setQuota.feature

Comment (saw-jan) [reply tests/acceptance/features/apiSpaces/spaceManagement.feature:93]: Please, add comment here with the links so that other can understand

Comment (PrajwolAmatya) [reply tests/acceptance/features/apiSpaces/spaceManagement.feature:93]: The deleted scenarios are covered on `setQuota.feature` file
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L94
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L134
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L50
- https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L14

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 2]
```diff
@@ -179,5 +179,27 @@ The expected failures in this file are from features in the owncloud/ocis repo.
...
+- [apiSpaces/setQuota.feature:174](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L174)
+- [apiSpaces/setQuota.feature:175](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L175)
+- [apiSpaces/setQuota.feature:176](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L176)
+- [apiSpaces/setQuota.feature:177](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L177)
+- [apiSpaces/setQuota.feature:178](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L178)
+- [apiSpaces/setQuota.feature:239](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L239)
+- [apiSpaces/setQuota.feature:240](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L240)
+- [apiSpaces/setQuota.feature:241](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L241)
```
Comment (amrita-shrestha) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:203]: Duplication of ### [API requests from an unauthorized user should return 403](https://github.com/owncloud/ocis/issues/5938) in line 136

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 3]
```diff
@@ -160,6 +160,27 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 - [apiGraph/removeUserFromGroup.feature:174](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/removeUserFromGroup.feature#L174)
 - [apiSpaces/createSpace.feature:18](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/createSpace.feature#L18)
 - [apiSpaces/createSpace.feature:19](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/createSpace.feature#L19)
+- [apiSpaces/setQuota.feature:80](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L80)
+- [apiSpaces/setQuota.feature:81](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L81)
+- [apiSpaces/setQuota.feature:82](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L82)
+- [apiSpaces/setQuota.feature:83](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L83)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:166]: why added is expected failure?

Comment (PrajwolAmatya) [reply tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:166]: When the space admin tries to set the quota of personal space of other users, then the HTTP status code returned is `401`, which is not the expected status code.

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md (hunk 3 above)
Comment (saw-jan) [reply tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:166]: Ahhh, I didn't notice the change in expected status code

**Review by saw-jan — COMMENTED** [review]
LGMT

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by grgprarup — COMMENTED** [review]
LGTM

**Comment by micbar** [comment]
I pushed a fix for the wrong status code.

**Review by micbar — APPROVED** [review]
Looks good now. 👍

**Pull request merged**